src/main/java/it/pn/frontend/e2e/common/HelpdeskPage.java

```diff
@@ -37,14 +35,14 @@ public HelpdeskPage(WebDriver driver) {
         super(driver);
     }
 
-    private final String helpdeskURL = "https://helpdesk.test.notifichedigitali.it/login";
 
-    public void changePage(){
-        this.driver.get(helpdeskURL);
+    public void changePage(String enviromentURL){
```

Comment:
@leleOFA rinomina environmentURL in HelpdeskURL

Reply:
do it